packages/E2ETest/wdio.conf.js

```diff
@@ -165,7 +165,8 @@ exports.config = {
   },
 
   jasmineNodeOpts: {
-    defaultTimeoutInterval: 60000,
+    // Needs to be longer than the 1m webdriverio timeout to let its errors be propgated
+    defaultTimeoutInterval: 100000,
```

Comment:
We were previously seeing a Jasmine timeout error if we went past the polling time on something instead of the correct error message